layouts/default.vue

```diff
@@ -5,5 +5,12 @@
     <div id="main">
       <slot />
     </div>
+    <NavigationLogin />
```

Comment:
small detail but I think "site" directory and SiteLogin fits better

Reply:
I agree. there is nothing about navigation in here. ->  SiteLoginLink makes sense imo.